requirements.txt

```diff
@@ -1,4 +1,4 @@
 colorlog == 6.6.0
 pydantic ~=1.9.2
 python-logstash ~= 0.4.8
-docker
\ No newline at end of file
+docker
```

Comment:
it is better to specify a version

Reply:
do we still this requirement, saw that you removed all docker related code

Reply:
https://github.com/ottopia-tech/keepalived-exporter/pull/5/commits/5b556e9425408ff5916e0b979b3f48f721a06de3